Reading List/BookController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  BookController.swift
+//  Reading List
+//
+//  Created by James Sedlacek on 6/5/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+class BookController: Codable {
```

Comment:
Great job with this class overall. Model controllers are not an easy thing to grasp at first, they take some time to get used to. One note, I don't think this class needs to conform to Codable for any reason.